backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -58,15 +54,15 @@ const parseWebhook = async (
   })
   // based on current setup, we expect the shape of body to be either
   // WhatsAppTemplateMessageWebhook or UserMessageWebhook
-  // if it's neither, we should thrown an error
+  // if it's neither, we should log an error and return early
   // ideally, should do full validation of the body using sth like Zod
   if (!body || typeof body !== 'object') {
     logger.error({
       message: 'Unexpected webhook body',
       action,
       body,
     })
-    throw new UnexpectedWebhookError('Unexpected webhook body')
+    return
   }
   if ('statuses' in body) {
     // can delete this after we verified that it all works
```

Comment:
There are a few of these comments in the file, do we still need them?

Reply:
Keep for traceability.

Reply:
Removing such comments is out-of-scope of this PR but sure, I've made a commit to remove them.